src/TurnoTablero.cpp

```diff
@@ -90,20 +88,21 @@ void TurnoTablero::concretarCambios(){
 		if (CambioARealizar->naceLaCelula()){
 			float factorNacimientoParcela = CambioARealizar->getParcela().getfactorNacimiento();
 			CambioARealizar->getParcela().getCelula()->nacer(factorNacimientoParcela, CambioARealizar->getColorPromedio());
-			this->celulasNacidasTurno ++;
+			//this->celulasNacidasTurno ++;
```

Comment:
a borrar esto ya lo hace internamente 
this->tableroAsociado->getDatosTablero()->sumarCelulaViva();